src/financial.py

```diff
@@ -0,0 +1,18 @@
+def calculate_compound_interest(principal, rate, periods):
+    """
+    Calculate compound interest on a monthly basis.
+
+    Args:
+        principal (float): The initial amount of money invested.
+        rate (float): The annual interest rate (expressed as a decimal).
+        periods (int): The number of periods the money is invested for.
+
+    Returns:
+        list: A list of values representing the amount after compound interest for each month.
+
+    """
+    amounts = []
```

Comment:
```suggestion
    amounts = [principal]
```
Sugiro a adição do montante inicial no gráfico, ainda mais que o primeiro ponto temporal do gráfico é a data atual (datetime.now), e o primeiro ganho de juros mensal só ocorre no fim do período do primeiro mês. Anteriormente, no datetime.now o montante já estava sendo incrementado instantâneamente.